portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase

```diff
@@ -1522,4 +1522,77 @@ definition {
 			value1 = "No identity provider is available to sign you in.");
 	}
 
+	@description = "LPS-125272 User profile matchup when NameID is transient, and screenName is in SAML Attributes list"
+	@priority = "4"
+	test ViewSPInitiatedSSOWithTransientNameId {
+		property test.name.skip.portal.instance = "SAML#ViewSPInitiatedSSOWithTransientNameId";
+		property portal.upstream = "gabor";
+
+		ApplicationsMenu.gotoPortlet(
+			category = "System",
+			panel = "Control Panel",
+			portlet = "Virtual Instances");
+
+		PortalInstances.addCP(
+			mailDomain = "liferay.com",
+			virtualHost = "www.able.com",
+			webId = "www.able.com");
+
+		// Setup localhost as IDP role
+
+		SAML.configureLiferaySAMLAsIdP(samlEntityID = "samlidp");
+
+		User.logoutPG();
+
+		// Create IDP - SP link, with transient name id and attribute maps containing screenName and emailAddress
+
+		SAML.configureLiferaySAMLAsSP(
+			spAttributes = '''screenName
+emailAddress
+firstName
+lastName
+uuid''',
+			nameIdentifierAttributeName = "static:CustomNameIDValue",
+			nameIdentifierFormat = "Transient",
+			samlEntityID = "samlsp",
+			idpAttributes = '''screenName=screenName
+emailAddress=emailAddress
+firstName=firstName
+lastName=lastName
+uuid=uuid''',
+			spURL = "http://www.able.com:8080");
```

Comment:
If order doesn't matter, sort the values of each variable and sort the order of the variables